docs/weblog/README.md

```diff
@@ -695,3 +695,10 @@ Query parameters required in the `GET` method:
 
 Examples:
 - `GET`: `/session/user?sdk_user=sdkUser`
+
+### \[GET\] /mongodb/getdatabase
+
+This endpoint is used to test if automatic instrumentation creates or not MongoDB spans depending of the configuration passed to the service.
```

Comment:
Do we need to add a new endpoint? Can we just enable/disable an integration and hit an existing endpoint

Reply:
Adding this new endpoint to all weblog apps would be a pretty time consuming task

Reply:
Updated [here](https://github.com/DataDog/system-tests/pull/3060/commits/5c5392d93919668df67e9ff4e921ff22aad14395) to make use of the existing `/kafka/produce` endpoint all languages seem to have but PHP(will need to figure out something else for it later but I already got a plan. 

The rest we should address as people start enabling if there are irregularities between languages in the test/spans.